packaging/os/pacman.py

```diff
@@ -261,6 +250,25 @@ def check_packages(module, pacman_path, packages, state):
         module.exit_json(changed=False, msg="package(s) already %s" % state)
 
 
+def expand_package_groups(module, pacman_path, pkgs):
+    expanded = []
+
+    for pkg in pkgs:
+        cmd = '%s -Sgq %s' % (pacman_path, pkg)
```

Comment:
Nitpicking here :)
The module uses `"` instead of `'` in creating cmd string in all places. We'd probably want to be consistent.